app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    elements = command.split(" ")
+    if elements[1] != elements[2]:
```

Comment:
* check that the command starts with `cp`
* check that such a file really exists `elements[1]`